go/libraries/doltcore/sqle/logictest/dolt/doltharness.go

```diff
@@ -57,6 +57,18 @@ func (h *DoltHarness) Init() error {
 		panic("Current directory must be a valid dolt repository")
 	}
 
+	return innerInit(h, dEnv)
+}
+
+func (h *DoltHarness) ExecuteStatement(statement string) error {
+	return executeStatement(h, statement)
```

Comment:
No reason to dispatch to these identical private methods, is there? Just test the public methods.

Reply:
+1